### PR TITLE
Add BC token json description

### PR DESCRIPTION
**Pull request by lailune**
Add token file for Block-Chain.com

### CODE REVIEW

File: tokens/eth/0x2ecb13a8c458c379c4d9a7259e202de03c8f3d19.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "symbol": "BC",
+  "name": "Block-Chain.com",
+  "type": "ERC20",
+  "address": "0x2ecb13a8c458c379c4d9a7259e202de03c8f3d19",
```
Comment (ligi) [inline tokens/eth/0x2ecb13a8c458c379c4d9a7259e202de03c8f3d19.json]:
```suggestion
  "address": "0x2ecB13A8c458c379c4d9a7259e202De03c8F3D19",
```

Comment (ligi) [reply tokens/eth/0x2ecb13a8c458c379c4d9a7259e202de03c8f3d19.json]: needs also to be done for the filename - but suggested changes do not yet cover this currently? cc @califa

**Comment by lailune** [comment]
Thanks. I fix It.

**Comment by ligi** [comment]
:+1:

**Pull request merged**